feature: preserve body after parse (#63)
openresty/lua-resty-upload: README.markdown

[Back to TOC](#table-of-contents)

Usage
=====

```lua
local upload = require "resty.upload"
local form, err = upload:new(self, chunk_size, max_line_size, preserve_body)
```

> `chunk_size` defaults to 4096. It will be the size used to read the socket.

Reviewer: ```suggestion
`chunk_size` defaults to 4096. It is the size used to read data from the socket.
```
